scripts/import_assets/import_assets.py

```diff
@@ -0,0 +1,85 @@
+import csv
```

Comment:
Please use `isort` on the file to sort the imports.

Reply:
Done!